### ISSUE TITLE
Thank you very much for your plugin. I was wondering if it would be possible to add a third step, which is to import the md document into the canvas to replace the original card. This plugin can provide links for cards in the canvas: https://github.com/Quorafind/Obsidian-Link-Nodes-In-Canva

### ISSUE THREAD

**Issue by CamWam** [slnsys/obsidian-canvas2document#5]
Thank you very much for your plugin. I was wondering if it would be possible to add a third step, which is to import the md document into the canvas to replace the original card. This plugin can provide links for cards in the canvas: https://github.com/Quorafind/Obsidian-Link-Nodes-In-Canvas

**Comment by slnsys** [comment]
Hello, to further evaluate your idea, could you please specify a bit more in detail? Which md document should get imported? The generated md documents (s. step 1 and step 2) are put next to the original canvas.

**Comment by slnsys** [comment]
oh, i got it now. You mean, referencing back from the new document to the originating canvas and its nodes? Like in this feature request (if it's not your's :-)
https://forum.obsidian.md/t/feature-request-for-link-to-a-block-in-a-note-within-canvas/81457

But it seams, when checking and testing the linking-syntax "[[...#...", "[[...^...", there is not yet support for the elements of canvases.

**Comment by slnsys** [comment]
yes, for its first conversion step canvas2document itself needs to convert all card nodes to embedded files. It does this for capsulating the structural content in these cards and have card level navigational headers to rearrange the doc.
So it already fulfills your requirements?

**Comment by slnsys** [comment]
Can look at it next week, as i'm on a journey. Canvas2document is able to convert everything in a canvas and it could get some extra functions for that.

**Comment by stdword** [comment]
@slnsys I'm here to request such `a third step, which is to import the md document into the canvas to replace the original card` too.

Use case: I need to share the one canvas card between several canvases without text duplicating.

Canvas card exists only within one canvas. I need the card content in .md and two or more canvases, that references that card via links to heading of MD doc.

**Comment by slnsys** [comment]
> [@slnsys](https://github.com/slnsys) I'm here to request such `a third step, which is to import the md document into the canvas to replace the original card` too.
> 
> Use case: I need to share the one canvas card between several canvases without text duplicating.
> 
> Canvas card exists only within one canvas. I need the card content in .md and two or more canvases, that references that card via links to heading of MD doc.

Sorry for the late reply.
So this is about generally converting card contents to MD file contents? And reimporting these MD files exactly to the location of the cards?
As a byproduct, canvas2document already converts all cards to MD files. So it wouldn't be too far off.

**Comment by stdword** [comment]
> And reimporting these MD files exactly to the location of the cards?

Yes! This is it!
